maintainers/scripts/travis-nox-review-pr.sh

```diff
@@ -1,58 +1,74 @@
 #! /usr/bin/env bash
-set -e
+set -ev
```

Comment:
All of the Travis examples have some it: https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps
I was hoping that Travis would minimize it? I'll remove it for now because it's not really related to the PR though.